cmake/CheckCXXLinkerFlag.cmake

```diff
@@ -54,7 +54,17 @@ set(check_cxx_linker_flag_patterns
 include (CheckCXXSourceCompiles)
 
 function(check_cxx_linker_flag _flag _var)
```

Comment:
🐛 see https://github.com/ROCm/MIOpen/commit/4549e017af33763b2751c316214399030e954f11#r144783324

Reply:
@BrianHarrisonAMD I've found it thanks to your previous comment 🤝

Reply:
Fixed now.